frontend_app_agenda/src/container/Agenda.jsx

```diff
@@ -227,7 +227,7 @@ class Agenda extends React.Component {
 
     const pageTitle = state.config.appConfig.idWorkspace === null
       ? props.t('All my agendas')
-      : props.t('Agenda of shared space {{workspaceLabel}}', {workspaceLabel: state.content.workspaceLabel})
+      : `${props.t('Agenda of shared space')} ${state.content.workspaceLabel}`
```

Comment:
Same as the title of page admin account, if you can fix the charset issue, you will can keep the translation key with variables (the `{{workspaceLabel}}`)